app/src/behaviors/behavior_sleep.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define DT_DRV_COMPAT zmk_behavior_sleep
+
+#include <device.h>
+#include <drivers/behavior.h>
+#include <logging/log.h>
+
+#include <zmk/behavior.h>
+#include <zmk/activity.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+static int behavior_sleep_init(const struct device *dev) { return 0; };
```

Comment:
Add a wrapper `#if HAS_DEVICE_STATUS_OKAY(DT_DRV_COMPAT)` like other drivers do now, so this doesn't compile if the DT is omitted.

Reply:
Added